Deprecated functions are used

### Checklist

- [x] I have used the search function for [**OPEN**](https://github.com/sabre-io/Baikal/issues) issues to see if someone else has already submitted the same bug report.
- [x] I have **also** used the search function for [**CLOSED**](https://github.com/sabre-io/Baikal/issues?q=is%3Aissue+is%3Aclosed) issues to see if the problem is already solved and just waiting to be released.
- [x] I will describe the problem with as much detail as possible.
- [x] This is not about the docker image. In that case, contact the authors of the docker image.

### Baikal version

0.11.1

### PHP version

8.5

### Steps to reproduce

Deprecated functions are used:

```code
------ ------------------------------------------------------------- 
  Line   Core\Frameworks\Baikal\Core\Server.php                       
 ------ ------------------------------------------------------------- 
  118    Call to deprecated method exec() of class Sabre\DAV\Server.  
         🪪  method.deprecated                                        
 ------ ------------------------------------------------------------- 

 ------ ------------------------------------------ 
  Line   Core\Frameworks\Flake\Controller\Cli.php  
 ------ ------------------------------------------ 
  40     Call to deprecated function strftime().   
         🪪  function.deprecated                   
  42     Call to deprecated function strftime().   
         🪪  function.deprecated                   
 ------ ------------------------------------------ 

 ------ ----------------------------------------- 
  Line   Core\Frameworks\Flake\Util\Tools.php     
 ------ ----------------------------------------- 
  245    Call to deprecated function strftime().  
         🪪  function.deprecated                  
  249    Call to deprecated function strftime().  
         🪪  function.deprecated                  
 ------ ----------------------------------------- 

 [ERROR] Found 5 errors                                                                                                
```

### Suggestion

- [x] add phpstan check
- [x] fix deprecation `strftime()`
- [x] fix deprecation `exec()`

--- a/Core/Frameworks/Baikal/Core/Server.php
+++ b/Core/Frameworks/Baikal/Core/Server.php
@@ -115,7 +115,7 @@ class Server {
      * @return void
      */
     function start() {
-        $this->server->exec();
+        $this->server->start();
     }
 
     /**

--- a/Core/Frameworks/BaikalAdmin/Resources/GlyphiconsPro/generate-sprite.php
+++ b/Core/Frameworks/BaikalAdmin/Resources/GlyphiconsPro/generate-sprite.php
@@ -456,7 +456,7 @@ CSS;
 CSS;
     }
 
-    $sCss = "\n/* " . count($aSprites) . " glyphs, generated on " . strftime("%Y-%m-%d %H:%M:%S") . "; C=" . $iCols . "; R=" . $iRows . "; W=" . $iPngWidth . "; H=" . $iPngHeight . "; PREFIX=" . $sClassPrefix . " */\n" . $sCss;
+    $sCss = "\n/* " . count($aSprites) . " glyphs, generated on " . (new DateTimeImmutable())->format('Y-m-d H:i:s') . "; C=" . $iCols . "; R=" . $iRows . "; W=" . $iPngWidth . "; H=" . $iPngHeight . "; PREFIX=" . $sClassPrefix . " */\n" . $sCss;
 
     return $sCss;
 }

--- a/Core/Frameworks/Flake/Controller/Cli.php
+++ b/Core/Frameworks/Flake/Controller/Cli.php
@@ -37,9 +37,9 @@ class Cli extends \Flake\Core\Render\Container {
         $this->sys_init();
         $this->init();
 
-        $this->echoFlush($this->notice("process started @" . strftime("%d/%m/%Y %H:%M:%S")));
+        $this->echoFlush($this->notice("process started @" . (new \DateTimeImmutable())->format('d/m/Y H:i:s')));
         $this->execute();
-        $this->echoFlush($this->notice("process ended @" . strftime("%d/%m/%Y %H:%M:%S")) . "\n\n");
+        $this->echoFlush($this->notice("process ended @" . (new \DateTimeImmutable())->format('d/m/Y H:i:s')) . "\n\n");
     }
 
     function execute() {

--- a/Core/Frameworks/Flake/Util/Tools.php
+++ b/Core/Frameworks/Flake/Util/Tools.php
@@ -241,14 +241,6 @@ class Tools extends \Flake\Core\FLObject {
         return strip_tags($sInput);
     }
 
-    static function getHumanDate($iStamp) {
-        return ucwords(strftime("%A, %d %B %Y", $iStamp));
-    }
-
-    static function getHumanTime($iStamp) {
-        return strftime("%Hh%M", $iStamp);
-    }
-
     static function trimExplode($string, $delim = ",", $removeEmptyValues = false, $limit = 0) {
         $explodedValues = explode($delim, $string);
 

--- a/composer.json
+++ b/composer.json
@@ -17,7 +17,8 @@
     },
     "require-dev" : {
         "friendsofphp/php-cs-fixer": "3.89.2",
-        "phpstan/phpstan": "~2.1.32"
+        "phpstan/phpstan": "~2.1.33",
+        "phpstan/phpstan-deprecation-rules": "~2.0.3"
     },
     "replace" : {
         "jeromeschneider/baikal" : "self.version"

--- a/phpstan.neon
+++ b/phpstan.neon
@@ -3,8 +3,8 @@ parameters:
   reportUnmatchedIgnoredErrors: false
   excludePaths:
     - Core/Frameworks/BaikalAdmin/Resources/GlyphiconsPro/generate-sprite.php
-    - Core/Resources/Web/BaikalAdmin/GlyphiconsPro/generate-sprite.php
-    - html/res/core/BaikalAdmin/GlyphiconsPro/generate-sprite.php
+    - Core/Resources/Web/BaikalAdmin/GlyphiconsPro/generate-sprite.php (?)
+    - html/res/core/BaikalAdmin/GlyphiconsPro/generate-sprite.php (?)
   ignoreErrors:
     -
       message: '#Instantiated class Flake\\Core\\Exception not found.#'
@@ -17,3 +17,6 @@ parameters:
       path: Core/Frameworks/Flake/Core/Model.php
     -
       identifier: unset.possiblyHookedProperty
+
+includes:
+    - vendor/phpstan/phpstan-deprecation-rules/rules.neon

